content/references/rippled-api/transaction-formats/transaction-types/trustset.md

```diff
@@ -31,9 +31,8 @@ Create or modify a trust line linking two accounts.
 | `LimitAmount`.`currency` | String    | (Amount.currency) | The currency to this trust line applies to, as a three-letter [ISO 4217 Currency Code](http://www.xe.com/iso4217.php) or a 160-bit hex value according to [currency format](currency-formats.html). "XRP" is invalid. |
 | `LimitAmount`.`value`    | String    | (Amount.value)    | Quoted decimal representation of the limit to set on this trust line. |
 | `LimitAmount`.`issuer`   | String    | (Amount.issuer)   | The address of the account to extend trust to. |
-| `QualityIn`              | Number    | UInt32            | _(Optional)_ Value incoming balances on this trust line at the ratio of this number per 1,000,000,000 units. A value of `0` is shorthand for treating balances at face value. |
-| `QualityOut`             | Number    | UInt32            | _(Optional)_ Value outgoing balances on this trust line at the ratio of this number per 1,000,000,000 units. A value of `0` is shorthand for treating balances at face value. |
-
+| `QualityIn`              | Number    | UInt32            | _(Optional)_ Value incoming balances on this trust line at the ratio of this number per 1,000,000,000 units. A value of `0` treats balances at face value. A value of `1` is recommended.  |
```

Comment:
A value of `1` is _NOT_ recommended.

I think you may be mixing up the notional idea behind this setting, and/or the [ripple-lib equivalent](https://xrpl.org/rippleapi-reference.html#trustline), with the actual functionality of this field.

This field is an integer which is considered to be a "per billion" rate. So a value of `1` here would mean that you value incoming balances at a rate of **0.0000001%**!

Reply:
To be clear, what _is_ recommended is not changing the value from its default of 0.

Reply:
Okay, great - thanks for clarifying! I got the value of 1 from this page, linked to in the issue: 

https://web.archive.org/web/20150422102043/https://wiki.ripple.com/Trust_line_quality

Reply:
Yeah, sorry about that. That's the only written doc we have of the thinking behind the feature, but it's pretty lacking, which is why we didn't just port it to the current dev portal.

The idea behind QualityIn/Out is that it represents a rate/percentage, but the way you specify it in the API is in the weird "integer billionths" format because that's more precise for tracking and storing in low-level data structures.